.circleci/config.yml

```diff
@@ -72,85 +64,13 @@ workflows:
-                gemfile: gemfiles/rails6.0_graphql1.11.gemfile
-              - ruby-version: '2.5'
-                gemfile: gemfiles/rails7.0_graphql2.0.gemfile
-              - ruby-version: '2.5'
-                gemfile: gemfiles/rails7.0_graphql2.1.gemfile
-              - ruby-version: '2.5'
-                gemfile: gemfiles/rails7.0_graphql2.2.gemfile
-              - ruby-version: '2.5'
-                gemfile: gemfiles/rails7.0_graphql2.3.gemfile
-              - ruby-version: '2.5'
-                gemfile: gemfiles/rails7.1_graphql2.0.gemfile
-              - ruby-version: '2.5'
-                gemfile: gemfiles/rails7.1_graphql2.1.gemfile
-              - ruby-version: '2.5'
-                gemfile: gemfiles/rails7.1_graphql2.2.gemfile
-              - ruby-version: '2.5'
-                gemfile: gemfiles/rails7.1_graphql2.3.gemfile
-              - ruby-version: '2.6'
-                gemfile: gemfiles/rails7.0_graphql1.11.gemfile
-              - ruby-version: '2.6'
-                gemfile: gemfiles/rails7.0_graphql1.12.gemfile
-              - ruby-version: '2.6'
-                gemfile: gemfiles/rails7.0_graphql1.13.gemfile
-              - ruby-version: '2.6'
-                gemfile: gemfiles/rails7.0_graphql2.0.gemfile
-              - ruby-version: '2.6'
-                gemfile: gemfiles/rails7.0_graphql2.1.gemfile
-              - ruby-version: '2.6'
-                gemfile: gemfiles/rails7.0_graphql2.2.gemfile
-              - ruby-version: '2.6'
-                gemfile: gemfiles/rails7.0_graphql2.3.gemfile
-              - ruby-version: '2.6'
-                gemfile: gemfiles/rails7.1_graphql2.0.gemfile
-              - ruby-version: '2.6'
-                gemfile: gemfiles/rails7.1_graphql2.1.gemfile
-              - ruby-version: '2.6'
-                gemfile: gemfiles/rails7.1_graphql2.2.gemfile
-              - ruby-version: '2.6'
-                gemfile: gemfiles/rails7.1_graphql2.3.gemfile
-              - ruby-version: '2.7'
-                gemfile: gemfiles/rails4.2_graphql1.8.gemfile
-              - ruby-version: '3.0'
-                gemfile: gemfiles/rails4.2_graphql1.8.gemfile
-              - ruby-version: '3.0'
-                gemfile: gemfiles/rails5.0_graphql1.8.gemfile
-              - ruby-version: '3.0'
-                gemfile: gemfiles/rails5.0_graphql1.9.gemfile
-              - ruby-version: '3.0'
-                gemfile: gemfiles/rails5.1_graphql1.8.gemfile
-              - ruby-version: '3.0'
-                gemfile: gemfiles/rails5.1_graphql1.9.gemfile
-              - ruby-version: '3.0'
-                gemfile: gemfiles/rails5.2_graphql1.8.gemfile
-              - ruby-version: '3.0'
-                gemfile: gemfiles/rails5.2_graphql1.9.gemfile
-              - ruby-version: '3.0'
-                gemfile: gemfiles/rails5.2_graphql1.10.gemfile
-              - ruby-version: '3.0'
-                gemfile: gemfiles/rails5.2_graphql1.11.gemfile
+              exclude:
```

Comment:
@HeyNonster indentation is 2 spaces too much to the right here, that's why CI is failing to run as the matrix is production way too many jobs

Reply:
Should be fixed now!